developer/src/kmc-copy/build.sh

```diff
@@ -29,29 +29,16 @@ builder_parse "$@"
 
 #-------------------------------------------------------------------------------------------------------------------
 
-do_test() {
-  # note: `export TEST_SAVE_ARTIFACTS=1` to save a copy of artifacts to temp path
-  # note: `export TEST_SAVE_FIXTURES=1` to get a copy of cloud-based fixtures saved to online/
-  local MOCHA_FLAGS=
-
-  if [[ "${TEAMCITY_GIT_PATH:-}" != "" ]]; then
-    # we're running in TeamCity
-    MOCHA_FLAGS="-reporter mocha-teamcity-reporter"
-  fi
-
-  eslint .
-  cd test
-  tsc --build
-  cd ..
-  readonly C8_THRESHOLD=70
-  c8 -skip-full --reporter=lcov --reporter=text --lines $C8_THRESHOLD --statements $C8_THRESHOLD --branches $C8_THRESHOLD --functions $C8_THRESHOLD mocha ${MOCHA_FLAGS} "${builder_extra_params[@]}"
-  builder_echo warning "Coverage thresholds are currently $C8_THRESHOLD%, which is lower than ideal."
-  builder_echo warning "Please increase threshold in build.sh as test coverage improves."
-}
+
 
 builder_run_action clean      rm -rf ./build/
 builder_run_action configure  verify_npm_setup
 builder_run_action build      tsc --build
 builder_run_action api        api-extractor run --local --verbose
-builder_run_action test       do_test
+
+# note: `export TEST_SAVE_ARTIFACTS=1` to save a copy of artifacts to temp path
+# note: `export TEST_SAVE_FIXTURES=1` to get a copy of cloud-based fixtures saved to online/
+# TODO: -skip-full
+builder_run_action test       builder_do_typescript_tests 70
```

Comment:
ok, and presumably we can crank the number up as tests improve..

Reply:
yes, you get a nice little warning that coverage is low. As tests are added, I have been bumping up the coverage number already. This is a little easier to read :grin: